src/models/TranslationValue/addTranslationValueToTranslationKey/index.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import TranslationValue from '@models/TranslationValue'
+import TranslationKey from '@models/TranslationKey'
+import Locale from '@models/Locale'
+import type { ValidationError } from '@types/ValidationError'
+
+export type AddTranslationValueToTranslationKeyArgs = {
+  value: string,
+  translationKeyId: string,
+  localeId: string,
+  ownerId: string
+}
+
+type AddTranslationValueToTranslationKeyResponse = {
+  translationKey?: TranslationKey,
+  errors: Array<ValidationError>
+}
+
+const addTranslationValueToTranslationKey = async ({
+  value,
+  translationKeyId,
+  localeId,
+  ownerId
+}: AddTranslationValueToTranslationKeyArgs): Promise<
+  AddTranslationValueToTranslationKeyResponse
+> => {
+  const errors: Array<ValidationError> = []
+
+  const translationKey: TranslationKey = await TranslationKey.query()
+    .findById(translationKeyId)
+    .eager('[project.locales, translationValues.locale]')
+
+  if (!translationKey) {
+    throw new Error('This translation key was not found.')
+  }
+
+  const project = translationKey.project
+  const locale = await Locale.query().findById(localeId)
+
+  if ((project && project.ownerId !== ownerId) || !locale) {
+    throw new Error('This project was not found.')
+  }
+
+  let isLocaleActivated = false
+  project.locales.forEach(projectLocale => {
+    if (projectLocale.id === locale.id) {
+      isLocaleActivated = true
+    }
+  })
+
+  if (!isLocaleActivated) {
+    throw new Error('This locale is not activated for this project')
+  }
+
+  if (value.length < 1) {
+    errors.push({
+      field: 'value',
+      message: 'Your value should be at least 1 character'
+    })
+
+    return { errors }
+  }
+
+  // Saving key
+  try {
+    const translationValue = new TranslationValue(value, locale)
+    translationKey.translationValues.push(translationValue)
```

Comment:
Same comment as in `src/models/TranslationKey/createTranslationKey/index.js`… Ideally we shouldn’t manipulate the JavaScript objects.